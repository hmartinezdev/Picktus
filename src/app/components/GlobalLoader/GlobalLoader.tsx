import Loader from '@components/Loader';
import colors from '@constants/colors';
import React, { Component } from 'react';

class Header extends Component {
  public render(): React.ReactElement<Header> {
    return (
      <div className="container">
        <div className="loaderContainer">
          <Loader />
        </div>

        <style jsx>{`
          .container {
            display: flex;
            justify-content: center;
            align-items: center;
            width: 8rem;
            height: 8rem;
            position: absolute;
            top: 50%;
            left: 50%;
            transform: translate3D(-50%, -50%, 0);
            z-index: 1000;
            width: 100%;
            min-height: 100%;
            background-color: ${colors.primary}cd;
          }

          .loaderContainer {
            width: 10rem;
            height: 10rem;
          }
        `}</style>
      </div>
    );
  }
}

export default Header;
