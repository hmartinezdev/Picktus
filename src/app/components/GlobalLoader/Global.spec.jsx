import { shallow } from 'enzyme';
import React from 'react';
import GlobalLoader from './GlobalLoader';

let wrapper;

const setup = () => shallow(<GlobalLoader />);

describe('<GlobalLoader />', () => {
  test('should render properly', () => {
    wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
