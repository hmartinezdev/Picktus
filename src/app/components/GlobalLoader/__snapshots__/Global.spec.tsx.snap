// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GlobalLoader /> should render properly 1`] = `
<div
  className="container"
>
  <div
    className="loaderContainer"
  >
    <Loader />
  </div>
  <style
    jsx={true}
  >
    
          .container {
            display: flex;
            justify-content: center;
            align-items: center;
            width: 8rem;
            height: 8rem;
            position: absolute;
            top: 50%;
            left: 50%;
            transform: translate3D(-50%, -50%, 0);
            z-index: 1000;
            width: 100%;
            min-height: 100%;
            background-color: #2d3047cd;
          }

          .loaderContainer {
            width: 10rem;
            height: 10rem;
          }
        
  </style>
</div>
`;
